components/ways-to-run/Hero.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <div class="c-hero center-text">
+    <h1 class="c-hero__header1">{{ $t('Ways to run your Dojo') }}</h1>
+    <div class="c-hero__intro">
+      <p>CoderDojo volunteers and clubs all around the world have been affected by the coronavirus pandemic. Champions and mentors from across the globe have talked to us to share how they are running their clubs now, and we're amazed by their resilience and creativity.</p>
+
+      <p>Inspired by these community examples, here are some ways you can continue to provide fun, creative learning opportunities for the young people and volunteers in your Dojo.</p>
+
+      <p>Choose the option that works best for you, depending on your current situation and your Dojo members' needs, and feel free to adapt your club activities if your situation and local health guidance change.</p>
+    </div>
+    <div class="c-hero__step-container">
+      <div class="c-hero__step-cards">
+         <div class="c-hero__step-card">
+            <img src="~/static/images/ways-to-run/ways-red.jpg" class="c-hero__step-image">
```

Comment:
Missing alt tag - can use `alt=""` as this is decorative image.
Check other images have correct alt tags too.

Reply:
Thanks v.much, they were on the to-do list but ended up getting swamped by other things :|

Reply:
Still missing? (Same with yellow and green?)